docs/docs/middlewares/path.md

```diff
@@ -0,0 +1,41 @@
+# Path
+This function returns the URL for the given route. 
+
+## Parameters
+```path(page, queries, locale)```
+
+parameter        | type           | required | description
+-----------------|----------------|----------|------------
+page             | string         | ✓        | The name of page (e.g. www.mysite.com).
+queries          | array          | ❌        | Array of url parameters (e.g. [gallery: nature])
+locale           | string         | ❌        | Shortcut for language.
+
+**Example**
+
+```
+# file.html.twig
+
+...
+<a href="{{ path('index') }}">
+    Back to homepage
+</a>
+...
+
+```
+
+**Example**
+
+This example shows how set absolute path in case of multilingual content of webpage.
```

Comment:
I would change it to: _This example shows how you can correctly link across multilingual versions of a page._

Reply:
And please provide more context in the code below, it should be obvious that this needs to be placed inside `<head>`.

Reply:
Easier to understand might be: This example shows how to easily create a link across multilingual versions of a current page.